src/coredump/coredumpctl.c

```diff
@@ -575,7 +575,7 @@ static int dump_list(sd_journal *j) {
         return 0;
 }
 
-static int save_core(sd_journal *j, int fd, char **path, bool *unlink_temp) {
+static int save_core(sd_journal *j, int fd, bool refuse_tty, char **path, bool *unlink_temp) {
```

Comment:
nitpick: `refuse_tty` sounds like this function would actually check for tty-ness itself. But it expects that the caller does it and only uses that flag to determine whether to show the error message. So maybe name it `show_tty_error`?

Still, this feels a bit weird. I see why it's done that way, but it feels like that whole thing should be rearranged. (Not necessarily in this PR, though).